package.json

```diff
@@ -10,9 +10,15 @@
     "temp-user": "NODE_ENV=testing npx sequelize-cli db:seed:all",
     "db-seed": "NODE_ENV=testing sequelize db:seed:all",
     "test": "NODE_ENV=testing npm run dropTables && NODE_ENV=testing npm run db-migrate && npm run db-seed && NODE_ENV=testing nyc --reporter=text --reporter=html --require @babel/polyfill --require @babel/register mocha ./src/tests/*  --timeout 200000 --exit",
-    "coverage": "nyc npm test  && npm run coveralls-coverage",
+    "coverage": "npm test && nyc report --reporter=text-lcov | coveralls",
     "dev-start": "nodemon --exec babel-node ./src/index.js",
     "create-db": "npx sequelize-cli model:generate --name users --attributes firstName:string,lastName:string,email:string,password:string,isVerified:string,signupType:string && sequelize db:migrate",
+    "create-db-users": "npx sequelize-cli model:generate --name users --attributes firstName:string,lastName:string,email:string,password:string,isVerified:string,signupType:string && sequelize db:migrate",
+    "create-db-status": "npx sequelize-cli model:generate --name status --attributes status:string && sequelize db:migrate",
+    "create-db-tripType": "npx sequelize-cli model:generate --name tripType --attributes tripType:string && sequelize db:migrate",
+    "create-db-trips": "npx sequelize-cli model:generate --name triprequest --attributes userId:integer,city:string,reason:string,tripType:string,status:string,startDate:string,returnDate:string",
+    "create-db-cities": "npx sequelize-cli model:generate --name cities --attributes name:string",
+    "create-db-triprequestcities": "npx sequelize-cli model:generate --name triprequestcities --attributes tripRequestId:integer cityId:integer",
```

Comment:
To avoid `package.json` to be complex, I recommend hiding all database scripts  in `.env` and use it when you want it.